runtime/lua/vim/lsp/protocol.lua

```diff
@@ -700,6 +700,9 @@ function protocol.make_client_capabilities()
     workspace = {
       symbol = {
         dynamicRegistration = false;
+        didChangeConfiguration =  {
+          dynamicRegistration = true;
```

Comment:
Not sure what this does, but I'm wondering if it's necessary for adding/removing workspace components. Can remove